index.html

```diff
@@ -8,8 +8,39 @@
 <div class="block">
   <h1><span class="header-inside">Projects</span></h1>
   <div id="projects-panel">
+    <table class="projects">
```

Comment:
Your `<table>` tags are outside of your `<noscript>` tags. As far as I can tell, this will cause issues - you should check to see if there's an empty set of `<table>`s if rendering the site with JS turned on.